sdk/core/Azure.Core/src/Pipeline/HttpClientTransport.cs

```diff
@@ -147,6 +143,46 @@ private static HttpClient CreateDefaultClient()
             };
         }
 
+        private static HttpMessageHandler CreateDefaultHandler()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("BROWSER")))
```

Comment:
This is wordy  but is the syntax that analyzer understands.

Reply:
Not exactly a hot path, but is this worth caching?

Reply:
Don't think so, it's a very non-hot path.